lib/ansible/modules/network/nxos/nxos_vrf.py

```diff
@@ -318,6 +322,14 @@ def map_obj_to_commands(updates, module):
                             commands.append('interface {0}'.format(i))
                             commands.append('no switchport')
                             commands.append('no vrf member {0}'.format(name))
+                elif interfaces and interfaces[0] == 'default':
```

Comment:
why check only the first item of the list to be default?

Reply:
See my reply above